src/test/java/com/wikia/webdriver/testcases/mercurytests/CategoryTest.java

```diff
@@ -16,19 +14,12 @@
 @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
 public class CategoryTest extends NewTestTemplate {
 
-  private Category category;
-
-  private void init() {
-    this.category = new Category(driver);
-
-    new Navigate(driver).toPage(MercurySubpages.MAIN_PAGE);
-  }
-
-  @Test(groups = "mercury_category_expandAndNavigateToCategoryPage")
-  public void mercury_category_expandAndNavigateToCategoryPage() {
-    init();
-
-    category.toggle();
-    category.openCategoryPage(0);
+  @Test(groups = "mercury_category_navigateToCategoryPageFromCategoryComponentOnMainPage")
+  public void mercury_category_navigateToCategoryPageFromCategoryComponentOnMainPage() {
+    new MainPage()
+        .openRegularMainPage()
+        .useCategoryComponent()
```

Comment:
The idea of having objects that represents page was to have user scenario and to focus on him.
We could have 3 objects that are used from dev perspective or we can focus and rethink, how we can do the same actions in 1 object and focus on user perspective.

Please leave it as it is, or find different name for `useCategoryComponent()` and `getCategoryComponent()` which will bring you mere context and name will reflect something that user do. Because when he navigates to main page he want to use catogory component to navigate to category page.